python/ray/train/examples/jax_mnist_example.py

```diff
@@ -0,0 +1,217 @@
+import argparse
```

Comment:
Is this from an existing JAX distributed example that we can link as a reference?